doc/ref/plotting_options/styling.ipynb

```diff
@@ -0,0 +1,519 @@
+   "metadata": {},
+   "source": [
+    "(option-c)=\n",
+    "## `c`\n",
+    "\n",
+    "Alias for [`color`](option-color) above."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3a1bc8fd-4667-483c-996a-d015da754cff",
+   "metadata": {},
+   "source": [
+    "(option-cmap)=\n",
+    "## `cmap`\n",
+    "\n",
+    "The `cmap` option controls the colormap used when mapping numerical or categorical data values to color. It supports:\n",
+    "- Named colormaps (e.g., 'viridis', 'plasma', 'coolwarm')\n",
+    "- Lists of color strings or hex codes (for custom sequences)\n",
+    "- Dictionaries (for categorical color mappings)\n",
+    "- Colormap objects from Matplotlib or HoloViews\n",
+    "\n",
+    "hvPlot selects a default colormap based on the data type, but cmap lets you override this behavior. Only one of `cmap`, `colormap`, or `color_key` should be used at a time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3c66bedd-6ec9-40ce-8c6b-2c5cf9526c5c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.earthquakes(\"pandas\")\n",
+    "\n",
+    "# Use a continuous colormap for magnitude\n",
+    "df.hvplot.scatter(\n",
+    "    x=\"lon\",\n",
+    "    y=\"lat\",\n",
+    "    c=\"mag\",\n",
+    "    cmap=\"plasma_r\",\n",
+    "    title=\"Earthquakes Colored by Magnitude\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1c34a309-d7c0-423e-adc7-9743f3d7a331",
+   "metadata": {},
+   "source": [
+    "(option-colormap)=\n",
+    "## `colormap`\n",
+    "\n",
+    "Alias for [`cmap`](option-cmap) above."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3290b419-10a3-4d2b-ac99-0e50c644fbcc",
+   "metadata": {},
+   "source": [
+    "(option-color_key)=\n",
+    "## `color_key`\n",
+    "\n",
+    "The `color_key` option defines a categorical color mapping, primarily used with `datashade=True`. It maps distinct categories in your dataset to specific colors.\n",
+    "\n",
+    "It accepts:\n",
+    "- A list of colors (matched to unique categories in order)\n",
+    "- A dictionary mapping category values to colors\n",
+    "\n",
+    "If not provided, hvPlot uses a default categorical colormap. Only one of `cmap`, `colormap`, or `color_key` should be used at a time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c7c13149-05de-4fcd-96eb-6bd6e84dea06",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.earthquakes(\"pandas\")\n",
+    "\n",
+    "# Define explicit color mapping for depth_class\n",
+    "depth_colors = {\n",
+    "    'Shallow': '#33ff58',\n",
+    "    'Intermediate': '#ffc300',\n",
+    "    'Deep': '#f74a64'\n",
+    "}\n",
+    "\n",
+    "df.hvplot.scatter(\n",
```

Comment:
Hm I wouldn't recommend users to use `color_key` in this case but `cmap` directly as it has the same effect (right?). I'd use `color_key` exclusively for plots with `datashade=True`. And to be honest, I'm not even sure to know why it's needed over `cmap`. Maybe you have that info in your colormapping notebook?

Reply:
It is mostly useful for datashaded plots but we don't have a large dataset within hvsampledata to use `datashade=True` so I just used it in a regular example.

To make it clearer, I can just add a note explaining the difference between them.